docker-deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
```

Comment:
I just updated it to build once with make build and then rebuild in the docker deploy script